client/src/Components/Store/ImageDisplay.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import sandClock from "../../Images/addTime.png"
+import eye from "../../Images/eye.png"
+import royal from "../../Images/royal.jpeg"
+import hallowween from "../../Images/hallowween.jpeg"
+
+const ImageDisplay = ({imgName, maxHeight, maxWidth}) => {
+    let imgsrc = ""
+
+    if (imgName == "Add Time") {
```

Comment:
Is this how you map the name to the image? So, `imgName == itemName` when passing it from the itemdata?

Reply:
Yes, I made the name more general incase I might need it again later on